response_operations_ui/templates/reset-password.html

```diff
@@ -1,132 +1,141 @@
-            {% set errorTitle = errorStrongerPasswordTitle %}
-            {% set errorDescription = errorStrongerPasswordDescription %}
-        {% elif errorType.password[0] == errorPasswordSameAsOldTitle %}
-            {% set errorTitle = errorPasswordSameAsOldTitle %}
-            {% set errorDescription = errorPasswordSameAsOldDescription %}
-        {% endif %}
-
-        {% call
-            onsPanel({
-                "type": "error",
-                "classes": "ons-u-mb-s",
-                "title": errorTitle
-            })
-        %}
-            <p>This <strong>must be corrected</strong> to continue.</p>
-            <a href="#inputPassword" class="ons-js-inpagelink">{{ errorDescription }}</a>
-        {% endcall %}
-    {% endif %}
-
-    <form
-        method="post"
-        action="{{ url_for('passwords_bp.post_reset_password', token=data.token) }}"
-        class="form ons-u-mt-xl"
-        role="form">
-        {{ form.csrf_token }}
-
-        <h1>Reset your password</h1>
-        <p>Your password must have:</p>
-        {{
-            onsList({
-                "itemsList": [
-                    {
-                        "text": "at least 12 characters"
-                    },
-                    {
-                        "text": "at least one capital letter"
-                    },
-                    {
-                        "text": "at least one symbol (eg ?!£%)"
-                    },
-                    {
-                        "text": "at least one number"
-                    }
-                ]
-            })
-        }}
-
-        {% if errorType.password %}
-            {% set error = { "text": errorType['password'][0] } %}
-        {% endif %}
-
-        {%- call onsFieldset({
-            "legend": "Choose a password",
-            "classes": 'ons-u-mt-l',
-            "legendClasses": "ons-u-vh",
-            "id": "reset-details"
-
-        }) -%}
-            {{
-                onsPassword({
-                    "id": "inputPassword",
-                    "name": "password",
-                    "label": {
-                        "text": "New password"
-                    },
-                    "showPasswordText": "Show password",
-                    "error": error
+    <div class="ons-grid">
+        <div class="ons-grid__col ons-col-8@m">
+        {% with messages = get_flashed_messages(category_filter=["error"]) %}
+            {% if messages %}
+                {% call
+                    onsPanel({
+                            "type": "error",
+                            "classes": "ons-u-mb-l",
+                            "title":  errorTitle
+                        })
+                %}
+                    {% for message in messages %}
+                        <p id="flashed-message-{{ loop.index }}">{{ message }}</p>
+                    {% endfor %}
+                {% endcall %}
+            {% endif %}
+        {% endwith %}
+          {% set password_error_count = 0 %}
```

Comment:
This was the repetition I was talking about

Reply:
As above. This has been removed.